doc/reference_accessor.xml

```diff
@@ -7,8 +7,7 @@
 	  <refnamediv>
 		<refname>GeometryType</refname>
 
-		<refpurpose>Returns the type of the geometry as a string. Eg:
-			'LINESTRING', 'POLYGON', 'MULTIPOINT', etc.</refpurpose>
+		<refpurpose>Returns the type of the geometry as text.</refpurpose>
```

Comment:
Small note about consistency (now that you are improving things quite a bit):
- Here it refers to the geometry as `the geometry`.
- In https://github.com/postgis/postgis/pull/399/commits/ef2b137635c4be8dbd1e9045b45a16ccacef0137#diff-f976dc5a4064695bcf05b055d1c2d912R93 as `the Geometry`.
- In https://github.com/postgis/postgis/pull/399/commits/ef2b137635c4be8dbd1e9045b45a16ccacef0137#diff-f976dc5a4064695bcf05b055d1c2d912R210 and https://github.com/postgis/postgis/pull/399/commits/ef2b137635c4be8dbd1e9045b45a16ccacef0137#diff-f976dc5a4064695bcf05b055d1c2d912R264 as `a Geometry`.
- In https://github.com/postgis/postgis/pull/399/commits/ef2b137635c4be8dbd1e9045b45a16ccacef0137#diff-f976dc5a4064695bcf05b055d1c2d912R280 as `this Geometry object`.

Should we care about this?

Reply:
given that this goes into COMMENT, can we lose articles and get it condensed so it's a bit easier to fit into screen?

Reply:
@Algunenano Yes. absolutely we should have consistency in referring to geometry, and also specific geometry types.  In aid of this I will create a PostGIS Doc style guide in the repo, as a place for discussion and to capture decisions.

For this particular issue, there's lots of options, but here's my current thinking:
* Refer to general geometry objects as "geometry"
* Refer to specific types as LineString, Polygon, etc
* Not sure about using a different font.  Not doing that is simpler.  At least should not be fonted in the summary text

Reply:
@Komzpa Yes, the comment should be shorter.  I will fix.

Reply:
@Komzpa Actually realized I already made this shorter.  Not so keen on removing all articles.  That hasn't been done elsewhere, and there are many longer comments.